src/neo/SmartContract/InteropService.Native.cs

```diff
@@ -8,12 +8,7 @@ partial class InteropService
         internal static class Native
         {
             public static readonly InteropDescriptor Deploy = Register("Neo.Native.Deploy", Native_Deploy, 0, TriggerType.Application, CallFlags.AllowModifyStates);
-
-            static Native()
-            {
-                foreach (NativeContract contract in NativeContract.Contracts)
-                    Register(contract.ServiceName, contract.Invoke, contract.GetPrice, TriggerType.System | TriggerType.Application, CallFlags.None);
-            }
+            public static readonly InteropDescriptor Call = Register("Neo.Native.Call", Native_Call, 0, TriggerType.System | TriggerType.Application, CallFlags.None);
```

Comment:
All TriggerTypes?

Reply:
We do not allow state access in verification trigger.